model/components/commercial/appdynamics.yml

```diff
@@ -0,0 +1,126 @@
+    link: https://docs.appdynamics.com/display/LATEST/AppDynamics+Application+Performance+Monitoring+Platform
+    twitter: AppDynamics
+    logo: appdynamics
+    categories:
+      - collector
+      - data-processing
+      - storage
+      - dashboarding
+      - alerting
+      - visualisation
+    capabilities:
+      os:
+        - linux
+        - Windows
+        - SaaS
+      tech:
+        - AWS
+        - Azure
+        - Google Cloudplattform
+      aspects:
+        - tracing
+        - metrics
+        - analytics    
+    connections:
+      dataTo:
+        - grafana
+      dataFrom:
+        - appdynamics-sdks
+        - appdynamics-agent
+        - opentracing    
+    isCommercial: true
+
+  # AppDynamics APM Agent
+  - id: appdynamics-agent
+    name: AppDynamics Agent
+    description: AppDynamics Application Performance Monitoring (APM), a component of the AppDynamics platform, gives you end-to-end visibility into the performance of your applications. The main goal of our agents is to have the lowest overhead possible on the system and during installation. Monst of the AppDynamics agents can be integrated without any code changes.
+    link: https://docs.appdynamics.com/display/LATEST/Application+Monitoring
+    twitter: AppDynamics
+    logo: appdynamics
+    categories:
+      - agent
+    capabilities:
+      os:
+        - linux
+        - windows
+        - MacOS
+      aspects:
+        - tracing
+        - metrics
+        - analytics
+        - eum
+      tech:
+        - java
+        - .net
+        - php
+        - python
+        - nodejs
+        - SAP ABAP
+        - IBM Integration Bus
+        - Webserver
+        - Browser Monitoring Java Script 
+    connections:
+      dataFrom:
+        - opentracing
+        - appdynamics-agent
+    isCommercial: true
+
+  # AppDynamics SDKs
+  - id: appdynamics-sdks
+    name: AppDynamics SDKs
+    description: AppDynamics Application Performance Monitoring (APM), a component of the AppDynamics platform, gives you end-to-end visibility into the performance of your applications. The main goal of our agents is to have the lowest overhead possible on the system and during installation. Monst of the AppDynamics agents can be integrated without any code changes.
+    link: https://docs.appdynamics.com/display/LATEST/Application+Monitoring
+    twitter: AppDynamics
+    logo: appdynamics
+    categories:
+      - instrumentation-lib
+    capabilities:
+      aspects:
+        - tracing
+        - metrics
+        - analytics
+      tech:
+        - java
+        - .net
+        - php
+        - python
+        - go 
+        - nodejs
+        - c++
+        - Amazon Lambda
+    connections:
+      dataFrom:
+        - opentracing
+        - appdynamics-agent
```

Comment:
Do you really have to use the SDK together with the Agent? Thus, is the SDK sending data first to the agent before it is forwarded to the controller through the agent?